package.json

```diff
@@ -42,12 +42,14 @@
   "dependencies": {
     "ansicolors": "^0.3.2",
     "bn.js": "^5.2.0",
+    "buffer": "6.0.3",
```

Comment:
Let's make this `^` since we aren't going to have auto upgrades on in this package. It's nice to pin deps in our repos but we don't want to in shared packages.